CI failure in integration tests due to traceback:

```
Traceback (most recent call last):
  File "/root/.ansible/tmp/ansible-tmp-1533747172.82-174198093792739/AnsiballZ_azure_rm_storageaccount.py", line 113, in <module>
    _ansiballz_main()
  File "/root/.ansible/tmp/ansible-tmp-1533747172.82-174198093792739/AnsiballZ_azure_rm_storageaccount.py", line 105, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File "/root/.ansible/tmp/ansible-tmp-1533747172.82-174198093792739/AnsiballZ_azure_rm_storageaccount.py", line 48, in invoke_module
    imp.load_module('__main__', mod, module, MOD_DESC)
  File "/tmp/ansible_azure_rm_storageaccount_payload_3fxkRz/__main__.py", line 456, in <module>
  File "/tmp/ansible_azure_rm_storageaccount_payload_3fxkRz/__main__.py", line 452, in main
  File "/tmp/ansible_azure_rm_storageaccount_payload_3fxkRz/__main__.py", line 173, in __init__
  File "/tmp/ansible_azure_rm_storageaccount_payload_3fxkRz/ansible_azure_rm_storageaccount_payload.zip/ansible/module_utils/azure_rm_common.py", line 987, in storage_models
  File "/tmp/ansible_azure_rm_storageaccount_payload_3fxkRz/ansible_azure_rm_storageaccount_payload.zip/ansible/module_utils/azure_rm_common.py", line 436, in log
AttributeError: 'AzureRMStorageAccount' object has no attribute 'module'
```

--- a/lib/ansible/module_utils/azure_rm_common.py
+++ b/lib/ansible/module_utils/azure_rm_common.py
@@ -984,7 +984,6 @@ class AzureRMModuleBase(object):
 
     @property
     def storage_models(self):
-        self.log('Getting storage models...')
         return StorageManagementClient.models("2017-10-01")
 
     @property

